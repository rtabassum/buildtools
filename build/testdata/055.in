load(":foo.bzl","foo",a="a",b="c","d")

load(":bar.bzl","bar")
load(
  ":foobar.bzl",foobar="foobar")
foo(bar())
