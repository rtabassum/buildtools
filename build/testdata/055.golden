load(":foo.bzl", "foo", "a", b = "c", "d")
load(":bar.bzl", "bar")
load(
    ":foobar.bzl",
    "foobar",
)

foo(bar())
